src/components/overview/sub-Components/imageCarousel/arrowImage.jsx

```diff
@@ -8,9 +8,9 @@ function DirectionalButton({ arrowDirection, icon, onImageArrowClick }) {
         onClick={() => {
```

Comment:
Glad you could use some of the ideas for my directional buttons in creating your image carousel!